web/modules/custom/solution/solution.module

```diff
@@ -421,14 +421,7 @@ function solution_build_related_solutions_list(array $ids) {
     $element = $view_builder->view($referenced_solution, 'view_mode_tile', $referenced_solution->language()->getId());
     $elements[$delta] = [
       '#type' => 'container',
-      '#attributes' => [
-        'class' => [
-          'listing__item',
-          'listing__item--tile',
-          'mdl-cell',
-          'mdl-cell--4-col',
-        ],
-      ],
+      '#extra_suggestion' => 'container_grid_item',
```

Comment:
Ok, I change type of this block from form_element to container and I added there the same markup as for other blocks.